app/features/ledger/Transfer.ts

```diff
@@ -57,6 +60,38 @@ const INS_REMOVE_BAKER = 0x14;
 const INS_UPDATE_BAKER_STAKE = 0x15;
 const INS_UPDATE_BAKER_RESTAKE_EARNINGS = 0x16;
 
+type Memo = string;
+
+/**
+ * Send memo to ledger for signing.
+ * N.B. This does not send the length of the memo, and the CBOR encoded string.
+ */
+async function sendMemo(
+    transport: Transport,
+    ins: number,
+    memo: Memo | undefined,
```

Comment:
If we type thing correctly, then this should not allow for an undefined value as far as I see it.